lib/ansible/modules/cloud/amazon/ecs_taskdefinition.py

```diff
@@ -166,6 +167,10 @@ def register_task(self, family, task_role_arn, network_mode, container_definitio
                     for port in ('hostPort', 'containerPort'):
                         if port in port_mapping:
                             port_mapping[port] = int(port_mapping[port])
+                    if network_mode == 'awsvpc' and 'hostPort' in port_mapping:
+                        if port_mapping['hostPort'] != port_mapping.get('containerPort'):
```

Comment:
Might want to cast port_mapping['hostPort'] to an int just in case they passed a string. At some point we could probably update this module to use argspec suboptions so we could ensure the ports are integers before the module uses them.

Reply:
That's actually done in the three preceding lines :)

Reply:
Oops. :)